lib/systems/platforms.nix

```diff
@@ -307,6 +340,43 @@ rec {
     uboot = null;
   };
 
+  scaleway-c1 = armv7l-hf-multiplatform // {
+    gcc = {
+      cpu = "cortex-a9";
+      fpu = "vfpv3";
+      float = "hard";
+    };
+  };
+
+  utilite = {
```

Comment:
@copumpkin It's been a long time since I got everything working, but you should find most of my notes in these files: https://github.com/elitak/nixos-utilite It can be quite tricky to get the thing booting with the default version of uboot they ship. The `ubootUtilite` attr in this commit will build a new image to replace the stock one, but I don't remember if it's essential. I think I just wanted it for uEnv.txt support. Good luck.